src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Quote.php

```diff
@@ -76,4 +77,13 @@ function (Mage_Sales_Model_Quote_Address_Total $total) {
             )
         );
     }
+    
+    /**
+     * @param Mage_Sales_Model_Quote $quote
+     * @return string
+     */
+    public function addCartPriceDisplaySettings(Mage_Sales_Model_Quote $quote)
+    {
+        $quote->setData(self::KEY_CART_PRICE_DISPLAY_SETTINGS, Mage::getStoreConfig('tax/cart_display/price'));
+    }
```

Comment:
Please use the constant Mage_Tax_Model_Config::XML_PATH_DISPLAY_CART_PRICE instead of the hardcoded string 'tax/cart_display/price'

Reply:
Done